packages/react-avatar/SPEC.md

```diff
@@ -177,83 +170,61 @@ export interface AvatarProps extends React.HTMLAttributes<HTMLElement> {
    * The color when displaying either an icon or initials.
    * * neutral (default): gray
    * * brand: color from the brand palette
-   * * colorful: picks a color from a set of pre-defined colors, based on a hash of the name prop.
-   *     If another method is needed to assign colors to avatars, use the colorIndex prop.
+   * * colorful: picks a color from a set of pre-defined colors, based on a hash of the name (or idForColor if provided)
+   * * [AvatarNamedColor]: a specific color from the theme
    *
    * @defaultvalue neutral
    */
-  colorVariant?: 'neutral' | 'brand' | 'colorful';
+  color?: 'neutral' | 'brand' | 'colorful' | AvatarNamedColor;
 
   /**
-   * When colorVariant="colorful", specifies which color in the list of avatar colors to use.
-   * If colorIndex is out of bounds of the colors list, it will wrap around.
-   *
-   * @defaultvalue calculated based on a hash of the name prop.
+   * Specify a string to be used instead of the name, to determine which color to use when color="colorful".
+   * Use this when a name is not available, but there is another unique identifier that can be used instead.
    */
-  colorIndex?: number;
-
-  /** Style tokens */
-  tokens?: AvatarTokenSet;
+  idForColor?: string;
```

Comment:
Wanted to call out this change to the original proposed API that was reviewed in the spec review.
* The `color` prop combines `colorVariant` and `colorIndex`, allowing a specific color to be named if needed.
* The `idForColor` prop replaces some of the functionality that `colorIndex` supported; allowing an alternate way of "randomly" assigning colors for avatars that don't have a name.